samples/append_rows_buffered.js

```diff
@@ -35,26 +34,24 @@ function main(
     const destinationTable = `projects/${projectId}/datasets/${datasetId}/tables/${tableId}`;
     const streamType = managedwriter.BufferedStream;
     const writeClient = new WriterClient({projectId: projectId});
-    const bigquery = new BigQuery({projectId: projectId});
 
     try {
-      const dataset = bigquery.dataset(datasetId);
-      const table = await dataset.table(tableId);
-      const [metadata] = await table.getMetadata();
-      const {schema} = metadata;
-      const storageSchema =
-        adapt.convertBigQuerySchemaToStorageTableSchema(schema);
-      const protoDescriptor = adapt.convertStorageSchemaToProto2Descriptor(
-        storageSchema,
-        'root'
-      );
-
       const streamId = await writeClient.createWriteStream({
         streamType,
         destinationTable,
       });
       console.log(`Stream created: ${streamId}`);
 
+      // Get table schema from WriteStream metadata.
+      const writeStream = await writeClient.getWriteStream({
```

Comment:
for buffered stream, you can get the table schema from CreateWriteStream's response?

Reply:
I made the mistake of making the `createWriteStream` method returning just the Write name/id. So we have two options here:
* Add another method that returns the full response - but can be confusing to have two methods with similar names.
* Change the current method to return the full response - basically a breaking change, but the library is flagged as EXPERIMENTAL, so we can do this without calling this a breaking change.

Reply:
Can we add fields to the current response? That shouldn't be breaking change?

Reply:
the current response is just a `string`

Reply:
A string of Status? If that's the case, let's introduce a new method of returning a struct. Maybe add FullResponse to the end of the name?

Reply:
done, added `createWriteStreamFullResponse` method